tools/ci/sql_v0.sql

```diff
@@ -157,6 +157,8 @@ CREATE TABLE IF NOT EXISTS `characters` (
   `organ_data` mediumtext NOT NULL,
   `rlimb_data` mediumtext NOT NULL,
   `nanotrasen_relation` varchar(45) NOT NULL,
+  `physique` varchar(45) NOT NULL,
+  `height` varchar(45) NOT NULL,
   `speciesprefs` int(1) NOT NULL,
```

Comment:
Do not add these to this file, and remove the edited insert

Reply:
done